docs/versioning-policy.md

```diff
@@ -0,0 +1,98 @@
+# Tobira Releases and Versioning Policy
+
+Tobira is released via [GitHub releases](https://github.com/elan-ev/tobira/releases).
+Every release contains a pre-built binary for Linux-systems and a full changelog.
+
+We currently only maintain one branch and thus only have one release track.
+In other words: we don't backport bugfixes or other patches.
+Only the latest version is officially supported.
+
+Tobira version identifiers look like `x.y-name`, e.g. `3.12-cherry`.
+
+- `x`: main version number. Is increased for releases that contain new features, major UI changes, or any other non-small change.
+- `y`: patch version number. Is increased for releases that only contain bugfixes and other very minor changes. Reset to 0 whenever `x` is increased.
+- `name`: an arbitrary nickname for this release. Is associated to `x` and changes if and only if `x` changes.
+
+**Important**: the version identifier carries no semantic information you can rely on!
+It is simply an identifier, that's easy to communicate and useful for "marketing".
+The rules for increasing the numbers are deliberately very underspecified and  just very roughly approximate the user-perceived "size" of the release.
+You must **not** rely on these numbers to derive any information about breaking changes or ease of updating.
+**This is not SemVer!**
+
+However, we do care a lot about providing you with all relevant information about breaking changes!
+Every release mentions all potentially breaking changes in its changelog.
+Additionally, there is a release overview with breaking changes [here](./releases.md).
+
+<br>
+<br>
+
+---
+
+## Breaking and non-breaking changes
+
+Our definition of these terms for Tobira. Defines what we guarantee and what not.
+
+### Breaking changes/stability guarantees
+
+We consider the following changes as *breaking*, highlighting them in changelogs and tracking them in [the release overview](./releases.md).
+The short tags at the beginning are the table column headers in said release overview.
+
+- *config*: Changing the configuration file in a backwards-incompatible way.
+- *auth*: Changing the auth-system in a backwards-incompatible way.
+- *cli*: Changing the stabilized parts of the CLI in a backwards-incompatible way.
+- *meili*: Requiring a new version of MeiliSearch.
+- *pg*: Dropping support for a non-EOL version of PostgreSQL.
+- *tmod*: Requiring a newer version of the Tobira module in Opencast.
+- *ocversion*: Requiring a new Opencast version.
```

Comment:
Once released, these will go hand in hand, and you cannot get a newer version of the module without getting a newer version of Opencast.

```suggestion
- *ocversion*: Requiring a new Opencast version.
```

Reply:
I actually started out with having them combined, but split them up in the last minute. I'm totally fine combining them.